Userland/Libraries/LibWeb/Layout/BlockContainer.cpp

```diff
@@ -12,11 +12,31 @@ namespace Web::Layout {
 BlockContainer::BlockContainer(DOM::Document& document, DOM::Node* node, NonnullRefPtr<CSS::StyleProperties> style)
     : Box(document, node, move(style))
 {
+    if (dom_node()) {
+        StringBuilder builder;
+        builder.append(dom_node()->class_name());
+        if (MUST(builder.to_string()) == "HTMLInputElement") {
+            // set_natural_width(MUST(CSS::Length(this->dom_node.size()), CSS : Length::Type::Ch).to_px(*this));
+            // set_natural_height(CSS::Length(1, CSS : Length::Type::Lh).to_px(*this));
+            dbgln("We found an input box type of BlockContainer - constructor 1");
```

Comment:
This looks like a debug message that was useful to you during development, but ask yourself if it's really something everyone needs to see going forward :)